ui/tests/acceptance/sync/secrets/overview-test.js

```diff
@@ -194,8 +193,8 @@ module('Acceptance | sync | overview', function (hooks) {
       this.server.post('/sys/activation-flags/secrets-sync/activate', (_, req) => {
         assert.strictEqual(
           req.requestHeaders['X-Vault-Namespace'],
-          'admin',
-          'Request is made to admin namespace'
+          undefined,
```

Comment:
i am fairly sure that this request _is_ supposed to be made to the `admin` namespace for HVD clusters since HVD users only have `admin` access (and not root level).

this https://github.com/hashicorp/vault/blob/d56f564362241a25972f511194ce42ad7988c73c/ui/lib/sync/addon/components/secrets/page/overview.ts#L77

Reply:
Ah, good catch. Yes this is the POST on activate that we do need to add the namespace. Got myself mixed up.